core/src/main/java/com/exadel/etoolbox/backpack/core/servlets/model/PathModel.java

```diff
@@ -52,9 +56,17 @@ public String getPath() {
 
     /**
      * Gets the whether or not child pages excluded from a package build
-     * @return String value
+     * @return boolean value
+     */
+    public boolean isIncludeChildren() {
```

Comment:
(naming) Why "isInclude"? You are not obliged to add "is..." to whatever Boolean-typed getter. Code must be read as text, and "is include something" is a grammatical error.

Reply:
done